ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -494,6 +494,19 @@ FCU_DFSS_DSP_HTDC_CHWDC:
   - HTDC
   - CHWDC
 
+FCU_DFSS_DFVSC_DSP_HTDC_CHWDC:
```

Comment:
why doesn't this implement `FDPM`, which uses the field referenced? that would completely define the type; if you are worried about some devices not having the filter, have two types that explicitly cover the entity fields.